cli/src/commands/diff.rs

```diff
@@ -52,38 +59,62 @@ pub(crate) struct DiffArgs {
     format: DiffFormatArgs,
 }
 
+fn resolve_revision(
+    workspace_command: &WorkspaceCommandHelper,
+    r: &Option<RevisionArg>,
+) -> Result<Commit, CommandError> {
+    workspace_command.resolve_single_rev(r.as_ref().unwrap_or(&RevisionArg::AT))
+}
```

Comment:
nit: maybe better to inline this as a closure? It's even shorter, and will probably help follow the code.

Reply:
Done